### DISCUSSION TITLE
Multi Card - Single Image

### DISCUSSION THREAD

**Discussion by JBrame** [prateekt/pokemon-card-recognizer#20] [Ideas]
This maybe in here somewhere already and I missed it, but is it possible, or could it be added, to find multiple cards in a single image?

**Comment by prateekt** [comment]
There is a branch boundary_detector where this functionality is currently being prototyped. The basic approach is to try to detect the card borders and then run the text-based classifier on the words found within the card borders to be able to detect/classify multiple cards in the same image. However, the card border detector has not yet been robustified. There might be better approaches.
